src/ServiceManagement/StorSimple/Commands.StorSimple/Constants.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple
+{
+    public class Constants
+    {
+        public const string RequestIdHeaderName = "x-ms-request-id";
```

Comment:
Created a new Class in the Common/Constants. Have a look at the naming convention and let us know if it is the right way